src/documentation/markdown/GettingStarted/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+## [5.67.0] - 2021-07-08
```

Comment:
I usually end up updating the changelog after merging the PR and before publishing it in npm, just in case the date or the number of the version are no longer the same by the time I deploy